queries.sql

```diff
@@ -175,3 +175,115 @@ HAVING COUNT(owners.full_name) = (SELECT MAX(NUMPETS)
     owner    | pets
 -------------+------
  Melody Pond |    3
+
+
+-- QUERIES WITH MANY TO MANY TABLES RELATIONSHIPS --
+
+SELECT animals.name AS ANIMAL, visits.date_of_visit, vets.name AS VET
+FROM animals INNER JOIN visits ON animals.id = visits.animal_id
+INNER JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'William Tatcher' AND visits.date_of_visit = (SELECT MAX(visits.date_of_visit)
+                            FROM vets INNER JOIN visits ON vets.id = visits.vet_id
+                            WHERE vets.name = 'William Tatcher');
+ animal  | date_of_visit |       vet
+---------+---------------+-----------------
+ Blossom | 2021-01-11    | William Tatcher
+(1 row)
+
+SELECT vets.name AS VET, COUNT(DISTINCT visits.animal_id) AS NUMANIMALS
+FROM visits INNER JOIN vets ON visits.vet_id = vets.id
+GROUP BY (vets.name)
+HAVING vets.name = 'Stephanie Mendez';
+       vet        | numanimals
+------------------+------------
+ Stephanie Mendez |          4
+(1 row)
```

Comment:
- [ ] <b>OPTIONAL:</b> The same thing goes here. Kindly always remember to comment out the results.